trade-finance-manager-ui/server/controllers/helpers/get-success-banner-message.helper.ts

```diff
@@ -9,9 +9,15 @@ import { isDealCancellationEnabled } from './deal-cancellation-enabled.helper';
  * @param userToken - the user token
  * @returns the success message to be shown or null
  */
-export const getScheduledCancellationBannerMessage = async ({ dealSnapshot, userToken }: { dealSnapshot: Deal; userToken: string }): Promise<string | null> => {
+export const getScheduledCancellationBannerMessage = async ({
+  dealSnapshot,
+  userToken,
+}: {
+  dealSnapshot: MappedDealSnapshot;
```

Comment:
I've amended this to take a new `MappedDealSnapshot` type - the snapshot passed in here is the mapped version of the database snapshot so previously the type was incorrect.